app/pages/messages/new/search_users.php

```diff
@@ -0,0 +1,20 @@
+<?php
+restrict_access();
+$search = strtolower(get_query_param("search", numeric: false));
```

Comment:
<img width="736" alt="image" src="https://github.com/user-attachments/assets/d0151339-1ec4-4e28-89bd-7fbe2931e46c">

got an error when typing something and deleting it.

Reply:
Ok fixed